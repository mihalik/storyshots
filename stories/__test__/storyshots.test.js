import initStoryshots from '../../src'
initStoryshots()
