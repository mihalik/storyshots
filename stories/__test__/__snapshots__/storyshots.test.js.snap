exports[`Storyshots Another Button with some emoji 1`] = `
<button
  onClick={[Function]}
  style={
    Object {
      "backgroundColor": "#FFFFFF",
      "border": "1px solid #eee",
      "borderRadius": 3,
      "cursor": "pointer",
      "fontSize": 15,
      "margin": 10,
      "padding": "3px 10px",
    }
  }>
  😀 😎 👍 💯
</button>
`;

exports[`Storyshots Another Button with text 1`] = `
<button
  onClick={[Function]}
  style={
    Object {
      "backgroundColor": "#FFFFFF",
      "border": "1px solid #eee",
      "borderRadius": 3,
      "cursor": "pointer",
      "fontSize": 15,
      "margin": 10,
      "padding": "3px 10px",
    }
  }>
  Hello Button
</button>
`;

exports[`Storyshots Button with some emoji 1`] = `
<button
  onClick={[Function]}
  style={
    Object {
      "backgroundColor": "#FFFFFF",
      "border": "1px solid #eee",
      "borderRadius": 3,
      "cursor": "pointer",
      "fontSize": 15,
      "margin": 10,
      "padding": "3px 10px",
    }
  }>
  😀 😎 👍 💯
</button>
`;

exports[`Storyshots Button with text 1`] = `
<button
  onClick={[Function]}
  style={
    Object {
      "backgroundColor": "#FFFFFF",
      "border": "1px solid #eee",
      "borderRadius": 3,
      "cursor": "pointer",
      "fontSize": 15,
      "margin": 10,
      "padding": "3px 10px",
    }
  }>
  Hello Button
</button>
`;

exports[`Storyshots Welcome to Storybook 1`] = `
<div
  style={
    Object {
      "fontFamily": "\"Helvetica Neue\", Helvetica, \"Segoe UI\", Arial, freesans, sans-serif",
      "lineHeight": 1.4,
      "margin": 15,
      "maxWidth": 600,
    }
  }>
  <h1>
    Welcome to STORYBOOK
  </h1>
  <p>
    This is a UI component dev environment for your app.
  </p>
  <p>
    We\'ve added some basic stories inside the
     
    <code
      style={
        Object {
          "backgroundColor": "#f3f2f2",
          "border": "1px solid #eae9e9",
          "borderRadius": 4,
          "color": "#3a3a3a",
          "fontSize": 15,
          "fontWeight": 600,
          "padding": "2px 5px",
        }
      }>
      src/stories
    </code>
     
    directory.
    <br />
    A story is a single state of one or more UI components. You can have as many stories as you want.
    <br />
    (Basically a story is like a visual test case.)
  </p>
  <p>
    See these sample
     
    <a
      href="#"
      onClick={[Function]}
      style={
        Object {
          "borderBottom": "1px solid #1474f3",
          "color": "#1474f3",
          "paddingBottom": 2,
          "textDecoration": "none",
        }
      }>
      stories
    </a>
     
    for a component called
     
    <code
      style={
        Object {
          "backgroundColor": "#f3f2f2",
          "border": "1px solid #eae9e9",
          "borderRadius": 4,
          "color": "#3a3a3a",
          "fontSize": 15,
          "fontWeight": 600,
          "padding": "2px 5px",
        }
      }>
      Button
    </code>
    .
  </p>
  <p>
    Just like that, you can add your own components as stories.
    <br />
    You can also edit those components and see changes right away.
    <br />
    (Try editing the 
    <code
      style={
        Object {
          "backgroundColor": "#f3f2f2",
          "border": "1px solid #eae9e9",
          "borderRadius": 4,
          "color": "#3a3a3a",
          "fontSize": 15,
          "fontWeight": 600,
          "padding": "2px 5px",
        }
      }>
      Button
    </code>
     component located at 
    <code
      style={
        Object {
          "backgroundColor": "#f3f2f2",
          "border": "1px solid #eae9e9",
          "borderRadius": 4,
          "color": "#3a3a3a",
          "fontSize": 15,
          "fontWeight": 600,
          "padding": "2px 5px",
        }
      }>
      src/stories/Button.js
    </code>
    .)
  </p>
  <p>
    This is just one thing you can do with Storybook.
    <br />
    Have a look at the
     
    <a
      href="https://github.com/kadirahq/react-storybook"
      style={
        Object {
          "borderBottom": "1px solid #1474f3",
          "color": "#1474f3",
          "paddingBottom": 2,
          "textDecoration": "none",
        }
      }
      target="_blank">
      React Storybook
    </a>
     
    repo for more information.
  </p>
</div>
`;
